lib/ReactTags.js

```diff
@@ -389,6 +400,7 @@ ReactTags.propTypes = {
   id: PropTypes.string,
   maxLength: PropTypes.string,
   inputValue: PropTypes.string,
+  tags: PropTypes.array,
```

Comment:
PropTypes.array.isRequired ?

Reply:
Is it required?

Reply:
yup its required thats why we have `[]` as default prop